content/getting-started/finding-kata.md

```diff
@@ -6,8 +6,44 @@ next: "/getting-started/solving-kata/"
 
 # Finding Kata
 
-<!---
-TODO training suggestions
-TODO kata search page
-TODO kata "numbers": rank, satisfaction, possibly others (which?) as criteria for selecting a kata to solve
--->
+Your account is configured and you are ready for your next challenge. Now you will learn how to find a task to train on.
+
+## Codewars trainer
+
+The easiest way of getting started is to use our trainer on the dashboard to pick your next kata.
+
+![training routines](./img/finding-kata_01_training-routines.png)
+
+The trainer allows you to pick your language, select your focus, preview the challenge, skip to another challenge and to start training.
+
+For newer developers, we recommend that you pick the "Fundamentals" focus track, which will randomly select easier kata for you to train on.
+
+For experienced developers looking for a challenge, you can select the "Rank Up" focus track, which will cause you to begin with easy to medium challenges and quickly advance to harder ones.
+
+If, for some reason, you do not like suggested kata, you can skip on it without any serious consequences. It will not be suggested to you by trainer anymore, but you will be always able to get back to it and solve it, for example by searching through kata library (see below).
+
+## Kata search page
+
+For those really looking for a hard challenge, we recommend you go directly to the full list of kata where you can select from over 6000 kata to train on. Remember, 1 kyū is the hardest level a kata can be.
```

Comment:
```suggestion
For those really looking for a hard challenge or for a specific kind of rank/task/..., we recommend you go directly to the full list of kata where you can select from over 8000 kata to train on. Remember, 8 kyū is the easiest level a kata can be.
```

* #74 about passive/possessive form again (I won't mark it in the next suggestions
* changed 1 kyu to 8 kyu, this way, still up to date if 1 dan katas are created one day... x)